integration/tests/economy/test_economics.py

```diff
@@ -780,3 +1013,162 @@ def test_call_contract_with_mapping_updating(
+        assert base_fee_per_gas == gas_price
+
+        recipient = w3_client.create_account()
+        transfer_value = 10
+
+        receipt = w3_client.send_tokens_eip_1559(
+            from_=account_with_all_tokens,
+            to=recipient,
+            value=transfer_value,
+            max_priority_fee_per_gas=0,
+            max_fee_per_gas=base_fee_per_gas,
+        )
+
+        assert w3_client.get_balance(recipient) == transfer_value
+
+        sol_balance_after = operator.get_solana_balance()
+        token_balance_after = operator.get_token_balance(w3_client)
+        sol_diff = sol_balance_before - sol_balance_after
+
+        assert sol_balance_before > sol_balance_after, "Operator SOL balance incorrect"
+        token_diff = w3_client.to_main_currency(token_balance_after - token_balance_before)
+        assert_profit(sol_diff, sol_price, token_diff, token_price, w3_client.native_token_name)
+        get_gas_used_percent(w3_client, receipt)
+
+    def test_eip_1559_too_low_fee(
+            self,
+            client_and_price: tuple[Web3Client, float],
+            operator: Operator,
+            account_with_all_tokens: LocalAccount,
+    ):
+        w3_client, token_price = client_and_price
+        sol_balance_before = operator.get_solana_balance()
+        token_balance_before = operator.get_token_balance(w3_client)
+
+        last_block = w3_client._web3.eth.get_block(block_identifier="latest")
+        base_fee_per_gas = last_block.baseFeePerGas  # noqa
+        gas_price = w3_client.gas_price()
+        assert base_fee_per_gas == gas_price
+
+        recipient = w3_client.create_account()
+        transfer_value = 10
+        too_low_max_fee = int(base_fee_per_gas * 0.8)
+
+        timeout = 10
+        native_token_name = w3_client.native_token_name
+
+        if native_token_name == "NEON":
+            exception = TimeExhausted
+            error_msg_regex = rf".+ not in the chain after {timeout} seconds"
+        else:
+            exception = ValueError
+            error_msg_regex = r"\{'code': -32000, 'message': 'transaction underpriced.+"
+
+        with pytest.raises(expected_exception=exception, match=error_msg_regex):
+            w3_client.send_tokens_eip_1559(
+                from_=account_with_all_tokens,
+                to=recipient,
+                value=transfer_value,
+                max_priority_fee_per_gas=0,
+                max_fee_per_gas=too_low_max_fee,
+                timeout=timeout,
+            )
+
+        assert w3_client.get_balance(recipient) == 0
+
+        sol_balance_after = operator.get_solana_balance()
+        token_balance_after = operator.get_token_balance(w3_client)
+        token_diff = token_balance_after - token_balance_before
+        sol_diff = sol_balance_before - sol_balance_after
+
+        assert token_diff == 0, "Operator NEON balance incorrect"
+        assert sol_diff == 0, "Operator SOL balance incorrect"
+
+    def test_eip_1559_profit_math(
```

Comment:
It is better to do this test with some iterative transaction (not simple transafer)